2-Authorization/1-call-own-api-aspnet-core-mvc/ToDoListClient/Views/Home/IndexModel.cshtml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Identity.Abstractions;
+using Microsoft.Identity.Web;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ToDoListClient.Models
+{
+    [AuthorizeForScopes(ScopeKeySection = "DownstreamApi:Scopes")]
+    public class IndexModel : PageModel
```

Comment:
I guess this is no longer a doc update? What is this change for?

Reply:
This is to enable the web app to display an error message when an exception occurs instead of crashing so that the signed in user has the opportunity to sign out. Recommendation from @jmprieur.

Reply:
actually, it might be better to move to a new PR

Reply:
https://github.com/Azure-Samples/ms-identity-ciam-dotnet-tutorial/pull/57